projects/Mallard/src/helpers/screen.ts

```diff
@@ -40,16 +43,19 @@ const convertImageSizeToImageDescription = (screenSize: number): ImageSize => {
     return (imageSize as ImageSize) || 'phone'
 }
 
-const imageForScreenSize = async () => {
+const IMAGE_SIZE_KEY = '@image_size'
+
+export const imageForScreenSize = async (): Promise<ImageSize> => {
+    const persisted = await AsyncStorage.getItem(IMAGE_SIZE_KEY)
+    const persistedSize = imageSizes.find(_ => _ == persisted)
+    if (persistedSize != undefined) {
+        return persistedSize
+    }
     const isTablet = await DeviceInfo.isTablet()
     const screenSize = isTablet ? maxScreenSize() : minScreenSize()
-    return convertImageSizeToImageDescription(screenSizeToImageSize(screenSize))
-}
-
-export {
```

Comment:
@AWare We have this style across the codebase so please keep consistent.

Reply:
We have both styles in active use, discussed with the team and there was a preference for `export const`